monkey/infection_monkey/transport/attack_telems/victim_host_telem.py

```diff
@@ -0,0 +1,21 @@
+from infection_monkey.transport.attack_telems.base_telem import AttackTelem
+
+__author__ = "VakarisZ"
+
+
+class VictimHostTelem(AttackTelem):
```

Comment:
Why does this exist? What's different here from the normal send machine information?